crates/eigenda/src/altda_commitment.rs

```diff
@@ -76,17 +74,25 @@ impl TryFrom<&[u8]> for AltDACommitment {
             return Err(AltDACommitmentParseError::UnsupportedDaLayerType);
         }
 
-        let versioned_cert = match value[2].try_into()? {
-            CertVersion::Version1 => {
+        let versioned_cert = match value[2] {
```

Comment:
not a big fan of these random indexing being spread all over the place. Think cleaner would be to index all of them at the beginning of the function and give them proper names?